test/support/decoding_pipeline.ex

```diff
@@ -0,0 +1,15 @@
+defmodule DecodingPipeline do
+  @moduledoc false
+
+  def get_options(%{in: in_path, out: out_path, pid: pid}) do
```

Comment:
I assume it's some kind of leftover from pipeline that used to be here, there's no point to use map right now

Reply:
`pid` - if changed to a normal function argument - can then have default `self()` value